util/ci.sh

```diff
@@ -16,6 +16,9 @@ case $1 in
       ruby -I lib bin/gem uninstall executable-hooks gem-wrappers bundler-unload -x --force -i "$gemhome@global"
     fi
 
+    # --force flag is a workaround for Windows executable conflict, root cause should be fixed
+    gem install rake --no-document --force
```

Comment:
I don't understand this. If is only for windows why will be installed for everything  and if the root cause "should be fixed" why is needed?

Reply:
I was writing an explanation but I decided to remove the commit instead since my goal here is to fix CI, and it was not necessary.